### ISSUE TITLE
Integrity protection; require confidentiality

### ISSUE THREAD

**Issue by wseltzer** [w3c/wot-security#61]
In 4.1.1 Secure Delivery and Storage of Thing Description, the document blurs the uses of authentication (assuring the message came from its intended sender) and integrity-protection (assuring message contents are completely and only what the sender intended). It should mention both. 

I'd include a stronger SHOULD for confidentiality everywhere. Even when no sensitive-seeming data is being exchanged, the possibility that sensitive data could be gleaned from the aggregate transmission or by its linkage to other communications is high (see [Pervasive monitoring is an attack](https://tools.ietf.org/html/rfc7258) ).

https://www.w3.org/TR/2017/NOTE-wot-security-20171214/#secure-delivery-and-storage-of-thing-description

**Comment by ereshetova** [comment]
>In 4.1.1 Secure Delivery and Storage of Thing Description, the document blurs the uses of >authentication (assuring the message came from its intended sender) and integrity-protection >(assuring >message contents are completely and only what the sender intended). It should mention >both.

I would argue that data authentication by definition includes data integrity since if attacker modifies the data (message contents) the sender of the message is not anymore just the original sender but he + attacker and usual data authenticity methods prevent that.  

>I'd include a stronger SHOULD for confidentiality everywhere. Even when no sensitive-seeming data is >being exchanged, the possibility that sensitive data could be gleaned from the aggregate transmission >or by its linkage to other communications is high (see Pervasive monitoring is an attack ).

Yes, I agree with this point, however, since this is a non-normative document, I am not quite sure on what is correct way to use the terms.

@mmccool Maybe we should think on consistent terminology for this document? Normally you have an option between MUST and SHOULD and it would be giving you the needed degree (when smth absolutely must be done and when smth is good to do but optional). But I guess here usage of MUST is not applicable, so how else can we show the "degree of recommendation"?

**Comment by mmccool** [comment]
TODO: Create a PR to put a normative SHOULD statement for confidentiality of TD distribution in the Thing Description document.

**Comment by mmccool** [comment]
This relates to Discovery, since it's about the mechanism for distribution of TDs.   It also relates to the definition of Authentication, etc. as in issue https://github.com/w3c/wot-security/issues/148

**Comment by mmccool** [comment]
I agree that we should probably have a strong statement (perhaps in Architecture) that any distribution mechanism for TDs should satisfy certain requirements... and one of these should be an ability to maintain confidentiality.   The Discovery document should then specify at least one mechanism that satisfies those high-level requirements.
